test/test.js

```diff
@@ -0,0 +1,1054 @@
+//During the test the env variable is set to test
+process.env.NODE_ENV = 'test'
```

Comment:
This can actually be added to the `package.json` file. Then the NODE_ENV will be set for all test files that run.